tests/integration/conftest.py

```diff
@@ -370,3 +414,73 @@ def is_mysql_ready():
         ),
     )
     await ops_test.model.wait_for_idle()
+
+
+@pytest.fixture(scope="module", name="test_image")
+def image_fixture():
```

Comment:
I don't see where this fixture is used

Reply:
This is used in `tests.integration.test_upgrade.test_create_example_blog`.